tests/wrapper/test_simulation_objects.py

```diff
@@ -118,7 +118,7 @@ def souc(traffic_update) -> SimulationObjectUpdatingComponent:
 def configured_souc(traffic_update) -> SimulationObjectUpdatingComponent:
     # pylint: disable=unused-argument
     return SimulationObjectUpdatingComponent(
-        sumo_configuration="sumo-config/example.scenario.sumocfg"
+        sumo_configuration="data/sumo/example/sumo-config/example.scenario.sumocfg"
```

Comment:
I recommend you to use path.join for this 😇 It differentiates between OS if you have to use `/` or `\`.